Tests/ttLib/data/TestTTF-Regular.ttx

```diff
@@ -468,7 +468,7 @@
       https://github.com/fonttools/fonttools
     </namerecord>
     <namerecord nameID="14" platformID="1" platEncID="0" langID="0x0" unicode="True">
-      https://github.com/fonttools/fonttools/blob/master/LICENSE
+      https://github.com/fonttools/fonttools/blob/main/LICENSE
```

Comment:
hm that's weird.. changing "master" -> "main" in this name string makes the `Tests/ttLib/woff2_test.py::WOFF2WriterTTFTest::test_hmtx_trasform` fail.. I'll take a look